stl/inc/optional

```diff
@@ -612,13 +612,14 @@ template <class _Ty>
 optional(_Ty) -> optional<_Ty>;
 
 template <class _Ty1, class _Ty2>
+_NODISCARD constexpr bool operator==(const optional<_Ty1>& _Left, const optional<_Ty2>& _Right) noexcept(

```

Comment:
Meh, these are just as ugly they way clang formatted them. Feel free to revert if you like, I have no strong feelings either way.

Reply:
I'm fine with this.